Ginger/GingerCoreNET/ActionsLib/WebAPI/ActWebAPIBase.cs

```diff
@@ -87,6 +87,9 @@ public new static partial class Fields
 
             public static string UseLegacyJSONParsing = "UseLegacyJSONParsing";
 
+            public static string KeyFilePath = "KeyFilePath";
+            public static string ImportKeyFile = "ImportKeyFile";
```

Comment:
Codacy found an issue: [Change the visibility of 'ImportKeyFile' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6345816)